steel/src/ethereum.rs

```diff
@@ -94,6 +94,7 @@ impl EvmBlockHeader for EthBlockHeader {
         // technically, this is only valid after EIP-4399 but revm makes sure it is not used before
         blk_env.prevrandao = Some(header.mix_hash);
         if let Some(excess_blob_gas) = header.excess_blob_gas {
+            // safe conversion: according to the consensus spec, excess_blob_gas is always an uint64
             blk_env.set_blob_excess_gas_and_price(excess_blob_gas.try_into().unwrap())
```

Comment:
Annoying that the upstream header type is factored to use ` u128`. I worry that an `unwrap` here would become a DoS vector, in that a malformed header could crash the host. I think we should use `as u64` instead, to simple truncate the result.
```suggestion
            // safe conversion: according to the consensus spec, excess_blob_gas is always an uint64
            blk_env.set_blob_excess_gas_and_price(excess_blob_gas as u64)
```

Reply:
I'm not sure, isn't a weird overflow even worse than a clean panic?
If I am a malicious host/RPC, there are so many ways I could make the guest panic, like sending wrong state root or corrupted inclusion proofs, ...